infra/modules/dns-config/main.tf

```diff
@@ -0,0 +1,72 @@
+# referencing the record that already exists in the console
+data "aws_route53_zone" "project_url" {
+  name = "wic-services.org"
+}
```

Comment:
Excuse my ignorance: Is it possible and recommended to manage Route53 in terraform? Or should the certificate stuff always be managed manually in the console?